docs/installation-and-operation/serialization.md

```diff
@@ -4,53 +4,229 @@ redirect_from:
 
-Only some artifacts are included in dumps and loads:
+## What gets exported
 
-- Collections (except for personal collections)
+Metabase only includes some artifacts its exports.
+
+- Collections (except for personal collections, unless specified by the `--user` flag)
 - Dashboards
 - Saved questions
+- Actions
+- Models
 - Segments and Metrics defined in the Data Model
+- SQL Snippets
 - Public sharing settings for questions and dashboards
 - Admin Panel settings, except for permissions
-- Database connection settings
+- Database connection settings 
 - Data Model settings
+- Events and timelines
+
+Metabase will export its artifacts to a directory of YAML files. The export includes:
+
+- A `settings.yaml` file that includes some basic, Metabase-wide settings
+- Directories that contain YAML files for various Metabase entities
+
+An example export could include the following directories, depending on what you exported:
+
+- actions
+- collections
+- databases
+
+In the `collections/cards` directory, you'll see that Metabase prefixes individual files with IDs to disambiguate entities that share the same name:
+
+```
+IA96oUzmUbYfNFl0GzhRj_accounts_model.yaml
+KUEGiWvoBFEc5oGQCEnPg_converted_customers.yaml
+qzNa8ZeFgFXrrIoF2g8m4_accounts_model_detail.yaml
+```
+
+## Example question
+
+Questions can be found in the `cards` directory of a collection directory. Here's an example card YAML file for a native question (a question written in SQL):
+
+```
+description: The number of plans by referral source.
+archived: false
+collection_position: null
+table_id: null
+result_metadata: null
+database_id: Sample Database
+enable_embedding: false
+collection_id: onou5H28Wvy3kWnjxxdKQ
+query_type: native
+name: Plan counts by source
+creator_id: jeff@metabase.com
+made_public_by_id: null
+embedding_params: null
+cache_ttl: null
+dataset_query:
+  type: native
+  native:
+    query: |-
+      SELECT count(*),
+             PLAN,
+             SOURCE
+      FROM accounts
+      WHERE SOURCE IS NOT NULL
+      GROUP BY PLAN,
+               SOURCE
+    template-tags: {}
+  database: Sample Database
+parameter_mappings: []
+serdes/meta:
+- model: Card
+  id: 17p_H8e2OpHGGJVxqg4sN
+  label: plan_counts_by_source
+display: table
+entity_id: 17p_H8e2OpHGGJVxqg4sN
+collection_preview: true
+visualization_settings:
+  table.pivot_column: PLAN
+  table.cell_column: COUNT(*)
+  column_settings: null
+parameters: []
+dataset: false
+created_at: '2023-05-22T14:32:28.124325'
+public_uuid: null
+```
+
+## Before exporting or importing
+
+If your Metabase is:
+
+- currently running, and
+- you're using the default H2 database
+
+You'll need to stop Metabase before exporting or importing.
+
+Otherwise, you're good.
+
+## Exporting a Metabase
+
+To export the contents of a Metabase instance, use the following command in your terminal:
```

Comment:
For later, but this is another very common question:

What directory should you run the export command in?
- If running locally
- If deployed on Docker (or other popular deployments)